### DISCUSSION TITLE
6.2.4 (New curve presentation, Grid effect reworks, Alarms into Monitoring)

### DISCUSSION THREAD

**Discussion by T-Troll** [T-Troll/alienfx-tools#160] [Announcements]
### Changes into this release:

Fan control GUI apps:
- New curve visualization style (for color-blind people) - now other sensor's curves have dashed style, and current sensor have wider line.
- Sensor/fan selection re-worked to avoid possible data loss at sensor/fan switch.
- Checking fan on/off for control now select this fan.
- Fan point drag zone size increased.
- Bug fixed: Rare exception, then sensor switched to the other.

`AlienFX-GUI`:
- New: "Reverse"  mode for grid effect implemented.
- Selecting "Fans and power" tab then fan control disabled doesn't provide question about fan detection anymore. Fan detection will be initiated automatically.
- Grid effect execution logic re-worked - now more precise and require fewer updates.
- "Profiles" tab modified:
  - New: Grid effects now can be copied across profiles/removed.
  - Profile zone settings checks now off by default to prevent accidental reset.
  - New profile fan control copy logic for copy and reset.
- Profile conversion from older version dialog box and code removed, see details above.
- Bug fixed: If fans detection fails, "Fans and power" tab activated instead of "Settings".
- Bug fixed: Grid effects doesn't work correctly for "Diagonal (right)" zone direction.

`AlienFX-Mon`:
- New: Simple alarm logic added (see [manual](https://github.com/T-Troll/alienfx-tools/blob/master/Doc/alienfx-mon.md) for details).
- UI re-worked, value reset button for selected sensor added.
- Bug fixed: Exception then trying to enable BIOS sensors control, but fan control can't start correctly.

Light control SDK:
- Internal zone structures reworked - now simple, smaller and faster.

Known issues:
- Windows 11 still a pain. Fan control can stop working after shutdown/restart for no reason. Under investigation.

<hr /><em>This discussion was created from the release <a href='https://github.com/T-Troll/alienfx-tools/releases/tag/6.2.4'>6.2.4 (New curve presentation, Grid effect reworks, Alarms into Monitoring)</a>.</em>

**Comment by MarlosLucas** [comment]
Hey T-Troll, regarding the W11 pain, I believe thats resumed by fast-boot enabled. Since I disabled it, the alienfx works solid and stable!

I think thats all...

**Comment by T-Troll** [comment]
I glad it helps you.
But, unfortunately, it's not the only issue - some people still can't load it, so i'll change loading soon.
